packages/block-library/src/navigation-menu/theme.scss

```diff
@@ -0,0 +1,7 @@
+.wp-block-navigation-menu {
+	.wp-block-navigation-menu-item {
+		.wp-block-navigation-menu-link {
+			color: inherit;
```

Comment:
Just checking...do you _need_ this much specificity here?

Reply:
Yes, since if not the colors defined by the inline styles won't be strong enough to be applied.